molo/profiles/templates/profiles/editprofile.html

```diff
@@ -3,29 +3,35 @@
 {% get_settings %}
 
 {% block content %}
-<div class="signin-form">
-	<div class="block turq">
-		<form method="post" id="frmProfileEdit" action="{% url 'molo.profiles:edit_my_profile' %}">{% csrf_token %}
-			<p>{% trans "Type new display name:" %}</p>
-			{{ form.alias }}
-      <p>{% trans "Update your date of birth:" %}</p>
-	      {{ form.date_of_birth }}
+<div class="profiles">
+	<form method="post" action="{% url "molo.profiles:edit_my_profile" %}" class="profiles__form">
+	{% csrf_token %}
+	<fieldset>
+		<label for="id_alias">{% trans "Type new display name:" %}</label>
+		{{ form.alias }}
+	</fieldset>
 
-			{% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-	      <p>{% trans "Update your mobile number:" %}</p>
-		      {{ form.mobile_number.errors }}
-		      {{ form.mobile_number }}
-			{% endif %}
-			{% if settings.profiles.UserProfilesSettings.show_email_field %}
-	      <p>{% trans "Update your email address:" %}</p>
-		      {{ form.email.errors }}
-		      {{ form.email}}
-			{% endif %}
-				<br>
-				<br>
-			<input type="submit" value='{% trans "Update profile" %}' class="btn">
-		</form>
-	</div>
-</div>
+	<fieldset>
+		<label for="id_date_of_birth_year">{% trans "Update your date of birth:" %}</label>
+		{{ form.date_of_birth }}
+	</fieldset>
+
+	{% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
+	<fieldset>
+		<label for="id_mobile_number">{% trans "Update your mobile number:" %}</label>
+		{{ form.mobile_number.errors }}
+		{{ form.mobile_number }}
```

Comment:
@Mitso @peterp can we add the hint for the `mobile number` and `email address` too